datastore/gcloud/aio/datastore/filter.py

```diff
@@ -113,8 +115,13 @@ def from_repr(cls, data: Dict[str, Any]) -> 'PropertyFilter':
         return cls(prop=prop, operator=operator, value=value)
 
     def to_repr(self) -> Dict[str, Any]:
-        return {
+        rep = {
             'op': self.operator.value,
-            'property': {'name': self.prop},
-            'value': self.value.to_repr(),
+            'property': {'name': self.prop}
         }
+        # Temporary workaround for handling arrayValue with PropertyFilter
```

Comment:
Could you please explain what's temporary about this?

Reply:
It looks we handle `to_repr` in 2 different ways for GQL and standard filter cases. The way we handle GQL case adds `arrayValue` by itself [here](https://github.com/talkiq/gcloud-aio/blob/master/datastore/gcloud/aio/datastore/value.py#L81) which looks better to me. Here I add a basic type check but It might be better to call appropriate one _maybe_  doing type inferring (I haven't investigated it deeply). That's the reason why I call this as "workaround"